tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv

```diff
@@ -447,3 +447,57 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureVirtualNetworkGatewayCommand","Set-AzVirtualNetworkGateway","0","1050","The parameter set 'RadiusServerConfiguration' for cmdlet 'Set-AzVirtualNetworkGateway' has been removed.","Add parameter set 'RadiusServerConfiguration' back to cmdlet 'Set-AzVirtualNetworkGateway'."
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureVirtualNetworkGatewayCommand","Set-AzVirtualNetworkGateway","0","1050","The parameter set 'Default' for cmdlet 'Set-AzVirtualNetworkGateway' has been removed.","Add parameter set 'Default' back to cmdlet 'Set-AzVirtualNetworkGateway'."
 "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.SetAzureVirtualNetworkGatewayCommand","Set-AzVirtualNetworkGateway","0","1050","The parameter set '__AllParameterSets' for cmdlet 'Set-AzVirtualNetworkGateway' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'Set-AzVirtualNetworkGateway'."
+"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.AddAzureExpressRouteCircuitAuthorizationCommand","Add-AzExpressRouteCircuitAuthorization","0","3000","The type of property 'RouteFilter' of type 'Microsoft.Azure.Commands.Network.Models.PSPeering' has changed from 'Microsoft.Azure.Commands.Network.Models.PSRouteFilter' to 'Microsoft.Azure.Commands.Network.Models.PSResourceId'.","Change the type of property 'RouteFilter' back to 'Microsoft.Azure.Commands.Network.Models.PSRouteFilter'."
```

Comment:
What are these breaking changes in PSRouteFilter?

I understand the removal of the InterfaceEndpoints feature, but is this related.  Breaking changes are not allowed.

Reply:
@markcowl type of RouteFilter property has been adjusted to reflect bugfix in Swagger. Please see discussion of this change with Cormac here: https://github.com/Azure/azure-powershell/pull/9250#discussion_r286584813

Reply:
got it, if the actually filled-in properties are the same, this is OK.